lib/req-res.js

```diff
@@ -126,7 +126,48 @@ module.exports = function ReqRes ( reqResOptions ) {
 			'data' : {}
 		} );
 
-		this.handle( callback );
+		this.handle( function ( message, respond ) {
+
+			callback( message, function ( data, msg, code ) {
+				/*
+					Support backward compatibility
+					will delete this as soon as all services will updated
+				 */
+				if ( data.status ) {
+					return respond( data );
+				}
+
+				/*
+					will later change this - after all the
+					services implement the new standard
+				 */
+				return {
+					'success' : function () {
+						respond( {
+							'status' : 'success',
+							'data'   : data
+						} );
+					},
+
+					'error' : function () {
+						respond( {
+							'status'  : 'error',
+							'message' : data, // interchanged because data is optional
+							'data'    : msg || {}, // optional
+							'code'    : code || 0 // optional
+						} );
+					},
+
+					'fail' : function () {
+						respond( {
+							'status' : 'fail',
+							'data'   : data
+						} );
+					}
+				};
```

Comment:
What happens if someone just calls `reply( 'my data' )` without doing a `.success()`, `.error()`, or `.fail()`? Should we throw some kind of an error?